geniza/corpus/management/commands/export_annotations.py

```diff
@@ -0,0 +1,143 @@
+import json
+import os
+from collections import defaultdict
+from urllib.parse import urlparse
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from geniza.annotations.models import Annotation, annotations_to_list
+from geniza.corpus.models import Document
+from geniza.footnotes.models import Footnote
+
+
+class Command(BaseCommand):
+    """Backup annotation data and synchronize to GitHub"""
+
+    v_normal = 1  # default verbosity
+
+    def handle(self, *args, **options):
+        if not getattr(settings, "ANNOTATION_BACKUP_PATH"):
+            raise CommandError(
+                "Please configure ANNOTATION_BACKUP_PATH in django settings"
+            )
+
+        annotations_output_dir = os.path.join(
+            settings.ANNOTATION_BACKUP_PATH, "annotations"
+        )
+        # define paths and ensure directories exist for compiled transcription
+        transcription_output_dir = {}
+        for output_format in ["txt", "html"]:
+            print([settings.ANNOTATION_BACKUP_PATH, "transcriptions", output_format])
+            format_path = os.path.join(
+                settings.ANNOTATION_BACKUP_PATH, "transcriptions", output_format
+            )
+            transcription_output_dir[output_format] = format_path
+            os.makedirs(format_path, exist_ok=True)
+
+        # identify content to backup based on documents with digital editions
+        docs = Document.objects.filter(
+            footnotes__doc_relation__contains=Footnote.DIGITAL_EDITION
+        ).distinct()
```

Comment:
Seems useful to have elsewhere, but it's such a small amount of code that it probably doesn't warrant a whole method.

Reply:
if we already had a custom document queryset it would make sense to add it there, but I'm not sure it's worth adding one for just this... IDK if we use this anywhere else